web/src/main/java/com/epam/rft/atsy/web/controllers/ApplicationStateController.java

```diff
@@ -84,7 +94,12 @@ public ModelAndView loadPage(@RequestParam Long applicationId,
           .build());
     }
 
+    ApplicationDTO applicationDTO = applicationsService.getApplication(applicationId);
```

Comment:
I would prefer to create a new CandidateService method which can return a candidate for a given application id.

That would save us the additional round trip to the database and back, what the `applicationService.getApplication(applicationId)` means now.